services/smartcards/writeAdminCard.py

```diff
@@ -12,8 +12,11 @@
 election_bytes = f.read()
 f.close()
 
-short_value = json.dumps(
-    {'t': 'admin', 'h': hashlib.sha256(election_bytes).hexdigest()})
+short_value = json.dumps({
+    't': 'admin',
+    'h': hashlib.sha256(election_bytes).hexdigest(),
+    'p': '000000',
```

Comment:
Unrelated to this PR, I noticed that the `writeAdminCard` script wasn't including a PIN either